pkg/volume/awsebs/aws_ebs.go

```diff
@@ -250,8 +250,9 @@ func getVolumeSource(
 
 func (plugin *awsElasticBlockStorePlugin) ConstructVolumeSpec(volName, mountPath string) (*volume.Spec, error) {
 	mounter := plugin.host.GetMounter(plugin.GetPluginName())
+	hu := plugin.host.(volume.KubeletVolumeHost).GetHostUtil()
```

Comment:
what happens if the casting fails?

Reply:
Good point, I will do this safely instead. =)